{
  "comments": [
    {
      "key": {
        "uuid": "9a3d3df0_d1cb8ca3",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 12,
      "author": {
        "id": 6527
      },
      "writtenOn": "2017-11-09T14:08:04Z",
      "side": 1,
      "message": "In addition to this, since  iRT!\u003deRT works the related subtests are enabled again",
      "revId": "bbe7b9569ad347f37b2b4119563e7a94c8927da9",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "9a3d3df0_a0eaf30c",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 12,
      "author": {
        "id": 6895
      },
      "writtenOn": "2017-11-10T11:10:15Z",
      "side": 1,
      "message": "no Thomas, it\u0027s still not working (i just tested it now). the tests at line 194-195 is still failing. do we have bug for this issue at ODL ?",
      "parentUuid": "9a3d3df0_d1cb8ca3",
      "revId": "bbe7b9569ad347f37b2b4119563e7a94c8927da9",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "9a3d3df0_f1d85081",
        "filename": "sdnvpn/test/functest/testcase_4.py",
        "patchSetId": 1
      },
      "lineNbr": 208,
      "author": {
        "id": 459
      },
      "writtenOn": "2017-11-09T12:24:24Z",
      "side": 1,
      "message": "https://jira.opnfv.org/browse/SDNVPN-187",
      "range": {
        "startLine": 208,
        "startChar": 14,
        "endLine": 208,
        "endChar": 64
      },
      "revId": "bbe7b9569ad347f37b2b4119563e7a94c8927da9",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}